.ci/static-checks.sh

```diff
@@ -460,6 +460,7 @@ static_check_license_headers()
 			--exclude="LICENSE" \
 			--exclude="*.md" \
 			--exclude="*.pb.go" \
+			--exclude="*pb_test.go" \
```

Comment:
Double checking it doesn't need a `.` as in `*.pb_test.go` for a accurate match.

Reply:
It does not the generated files are `src/runtime/virtcontainers/pkg/agent/protocols/grpc/healthpb_test.go` and `src/runtime/virtcontainers/pkg/agent/protocols/grpc/ocipb_test.go`